utils/linux-browser-dependencies/inside_docker/process.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+set +x
+
+# Install Node.js
+
+apt-get update && apt-get install -y curl && \
+    curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
+    apt-get install -y nodejs
+
+# Install apt-file
+apt-get update && apt-get install -y apt-file && apt-file update
```

Comment:
I believe it'd be better if we run it as a normal user. Not sure if it'd make any difference in this case but the library search paths under root are usually slightly different from those of non-privileged users.

Reply:
Discussed it offline and decided to land as-is and iterate if needed.